Library/Formula/hue.rb

```diff
@@ -0,0 +1,157 @@
+class Hue < Formula
+  desc "Hue is a Web interface for analyzing data with Apache Hadoop."
+  homepage "http://gethue.com/"
+  url "https://github.com/cloudera/hue/archive/release-3.8.1.tar.gz"
+  # version "3.8.1" # redundant with version scanned from URL
+  sha256 "582777f567b9f4a34e4ce58cfd5ed24aff15f3e02f193e7990329d8cb0161a8a"
+
+  # system dependencies - either Homebrew or Mac OS versions
+  depends_on :java
+  depends_on :python
```

Comment:
Adding a `:python` dependency introduces a hard dependency on Homebrew python for bottles. If it's not important which Python this is built against, please remove the `:python` dependency so it will always be built against system Python.